### PR TITLE
[Bugfix] Stats JS links pointing to old applications show path

### PR DESCRIPTION
**Pull request by Martouta**
fixes [THREESCALE-1366](https://issues.jboss.org/browse/THREESCALE-1366)
With fake data, now the javascript has the service id:
image
image

### CODE REVIEW

File: assets/stats/lib/application_details.es6 [hunk 1]
```diff
@@ -3,12 +3,13 @@ export const applicationDetails = function (data) {
     return {
       id: data.id,
       name: data.name,
-      link: createLink(data.id, type)
+      link: createLink(data, type)
```
Comment (Martouta) [inline assets/stats/lib/application_details.es6]: This comes from this stacktrace:
https://github.com/3scale/porta/blob/4e12fc08a135de509ad57683ae082a75fc5cfb2d/assets/stats/lib/application_details.es6#L1-L2
https://github.com/3scale/porta/blob/4e12fc08a135de509ad57683ae082a75fc5cfb2d/assets/stats/provider/stats_top_apps.es6#L34-L37
https://github.com/3scale/system/blob/3213fb8ef08e86df86ccf837517d30b3f446fd4c/assets/stats/lib/series.es6#L37
https://github.com/3scale/system/blob/3213fb8ef08e86df86ccf837517d30b3f446fd4c/assets/stats/lib/series.es6#L16-L18
By writing `debugger` on the last one and faking data with these steps:
1. `rake backend:fake PROVIDER_ID=2`
2. `rake jspm:dev`
3. `rake jspm:dev:server`
I could see what it received and from where 😄 Thanks @didierofrivia  💪

Comment (Martouta) [reply assets/stats/lib/application_details.es6]: This comes from this stacktrace:
https://github.com/3scale/porta/blob/4e12fc08a135de509ad57683ae082a75fc5cfb2d/assets/stats/lib/application_details.es6#L1-L2

https://github.com/3scale/porta/blob/4e12fc08a135de509ad57683ae082a75fc5cfb2d/assets/stats/provider/stats_top_apps.es6#L34-L37

https://github.com/3scale/system/blob/3213fb8ef08e86df86ccf837517d30b3f446fd4c/assets/stats/lib/series.es6#L37

https://github.com/3scale/system/blob/3213fb8ef08e86df86ccf837517d30b3f446fd4c/assets/stats/lib/series.es6#L17

By writing `debugger` on the last one and faking data with these steps:
1. `rake backend:fake PROVIDER_ID=2`
2. `rake jspm:dev`
3. `rake jspm:dev:server`
I could see what it received and from where 😄 Thanks @didierofrivia  💪

File: app/lib/stats/views/usage.rb [hunk 2]
```diff
@@ -36,6 +36,9 @@ def usage(options)
             :account => {
               :id    => @cinstance.user_account.id,
               :name  => @cinstance.user_account.org_name
+            },
+            service: {
+              id: @cinstance.service.id
```
Comment (hallelujah) [inline app/lib/stats/views/usage.rb]: Can't we use `service_id`

**Comment by Martouta** [comment]
Thanks guys 💃

**Pull request merged**